op_crates/crypto/Cargo.toml

```diff
@@ -16,4 +16,7 @@ path = "lib.rs"
 [dependencies]
 deno_core = { version = "0.77.1", path = "../../core" }
 rand = "0.8.2"
-
+ring = "0.16.19"
+tokio = { version = "1.1.1", features = ["full"] }
+serde_json = { version = "1.0.61", features = ["preserve_order"] }
+serde = { version = "1.0.121", features = ["derive"] }
```

Comment:
Why are all those dependencies added here? `serde` and `serde_json` are reexported from `deno_core` anyway, so no need to declare them here

Reply:
@bartlomieju All other op_crates include `serde` as a dependency. Why would this one be any different?
If they shouldn't, I'd be happy to open a PR to remove them.